src/designsystem/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Image from "next/image";
+
+interface AvatarProps {
+  imgSrc: string;
+  alt: string;
+  size?: "small" | "medium" | "large";
+}
+
+export const Avatar = ({ imgSrc, alt, size, ...props }: AvatarProps) => {
```

Comment:
nitpick: you're not using `...props` so I don't think it should be included here